blog/2024-04-05-translation-key-filtering.md

```diff
@@ -0,0 +1,17 @@
+---
+author: Joost Lekkerkerker
+authorURL: https://github.com/joostlek
+authorImageURL: https://avatars.githubusercontent.com/u/7083755?v=4
+title: "Changing the way translation keys are handled"
+---
+
+In Home Assistant Core 2024.4.2, we are changing the way translation keys are downloaded.
```

Comment:
I found "downloaded" here is confusing. Download from where to where?
How about:
> In Home Assistant Core 2024.4.2, we are changing the way translations are included into builds from Lokalise.